lib/suspenders/app_builder.rb

```diff
@@ -366,6 +372,14 @@ def create_heroku_apps(flags)
       create_production_heroku_app(flags)
     end
 
+    def create_heroku_pipeline
+      pipeline_name = app_name.dasherize
+
+      run_heroku "pipelines:create -a #{app_name.dasherize}-staging", "staging"
```

Comment:
Hi @zamith, I've used it without the ```-staging``` suffix with no problem. If you check the Pipelines docs on Heroku, they don't either. Maybe this slipped in by mistake as they do use ```-staging``` in one of their commands when adding an app to a pipeline.